src/ploigos_step_runner/step_implementers/sign_container_image/curl_push.py

```diff
@@ -130,18 +132,20 @@ def _run_step(self):
                     container_image_signature_name=container_image_signature_name,
                     signature_server_url=signature_server_url,
                     signature_server_username=signature_server_username,
-                    signature_server_password=signature_server_password
+                    signature_server_password=signature_server_password,
+                    with_fips=with_fips
                 )
 
             step_result.add_artifact(
                 name='container-image-signature-url', value=container_image_signature_url,
             )
-            step_result.add_artifact(
-                name='container-image-signature-file-md5', value=signature_file_md5,
-            )
-            step_result.add_artifact(
-                name='container-image-signature-file-sha1', value=signature_file_sha1
-            )
+            if not with_fips:
+                step_result.add_artifact(
+                    name='container-image-signature-file-md5', value=signature_file_md5,
```

Comment:
Just wanted to double check no downstream steps depend on this result artifact as you are now suppressing it if fips.

Reply:
there isn't. it was just for funzy that i recorded them.